Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml

```diff
@@ -110,15 +122,23 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
   "18":
     id: "18"
-    taskid: 8189698a-475c-433b-8fbf-20275b3b2084
+    taskid: 6b49c26d-0f94-4a7d-82f3-cb91ec5d4530
     type: playbook
     task:
-      id: 8189698a-475c-433b-8fbf-20275b3b2084
+      id: 6b49c26d-0f94-4a7d-82f3-cb91ec5d4530
       version: -1
       name: File Enrichment - Generic v2
-      description: ""
+      description: |-
+        Enrich a file using one or more integrations.
+
```

Comment:
need to add to the original sub-playbook that we are using the following command there
`* File Reputation using !file`

Reply:
done